src/app/pages/administration/url-checker/url-checker.jsx

```diff
@@ -0,0 +1,336 @@
+                        <CardContent>
+                          {urlCheckResponse.httpResponseAssertion?.actualValue ? (
+                            <>
+                              <Typography variant="h6" style={{ fontWeight: 600 }}>
+                                HTTP Status Code:
+                              </Typography>
+                              <Box className={classes.statusText}>
+                                <Typography>
+                                  {urlCheckResponse.httpResponseAssertion.actualValue}
+                                </Typography>
+                                {urlCheckResponse.httpResponseAssertion.passed
+                                  ? (
+                                    <CheckCircleIcon fontSize="large" style={{ color: 'green', marginLeft: '8px', marginTop: '4px' }} />
+                                  ) : (
+                                    <CancelIcon fontSize="large" style={{ color: 'red', marginLeft: '8px', marginTop: '4px' }} />
+                                  )}
+                              </Box>
+                              <Typography variant="body2">
+                                <a
+                                  href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status"
+                                  target="_blank"
+                                  rel="noopener noreferrer"
+                                >
+                                  Reference for HTTP Status Codes
+                                </a>
+                              </Typography>
+                            </>
+                          ) : (
+                            <>
+                              <Typography variant="h6" style={{ fontWeight: 600 }}>
+                                No HTTP Status Code Available:
+                              </Typography>
+                              <Box className={classes.statusText}>
+                                <Typography>
+                                  The HTTP response code could not be retrieved or is unavailable.
+                                </Typography>
+                                {urlCheckResponse.httpResponseAssertion.passed
+                                  ? (
+                                    <CheckCircleIcon fontSize="large" style={{ color: 'green', marginLeft: '8px', marginTop: '4px' }} />
+                                  ) : (
+                                    <CancelIcon fontSize="large" style={{ color: 'red', marginLeft: '8px', marginTop: '4px' }} />
+                                  )}
+                              </Box>
+                            </>
+                          )}
+                        </CardContent>
+                      </Card>
+                      <Card className={classes.resultsCard} style={{ zIndex: 2 }}>
+                        <CardContent>
+                          <Typography variant='h6' style={{ fontWeight: 600 }}>
```

Comment:
```suggestion
                          <Typography variant="h6" style={{ fontWeight: 600 }}>
```

The linter should have picked this up